### ISSUE TITLE
Uncaught TypeError: _plainDraggableDefault.default is not a constructor

### ISSUE THREAD

**Issue by romanzimnik** [anseki/plain-draggable#98]
First of all: plain-draggable is great stuff (thanks for the work on this) and exactly what I need for a project!

I've been trying to use plain-draggable and integrate it into my project for days now, but never get past this error message:

```
Uncaught TypeError: _plainDraggableDefault.default is not a constructor
    at Object.3P6l9.@parcel/transformer-js/lib/esmodule-helpers.js (index.63f9b973.js:441)
    at newRequire (index.63f9b973.js:71)
    at index.63f9b973.js:120
    at index.63f9b973.js:143
```

My setup is currently a maximally simple one: an npm-based project, a simple HTML-file with some SVGs and a js-file into which I try to import PlainDraggable as follows: 

```
import PlainDraggable from "plain-draggable";

let draggable = new PlainDraggable(document.getElementById('draggable'));
```

I am trying to start it by using parcel with `npx parcel index.html`. Works great with every other script I tried to integrate.

I tried a lot and even watched whole tutorials on how to import libraries in JavaScript (I'm pretty new to JS, never really worked intensively with JS and would need some help here).

Thanks in advance for a little advice!

**Comment by anseki** [comment]
Hi @romanzimnik, thank you for the comment.
I don't know the `_plainDraggableDefault` symbol, therefore your framework (or something) did something incorrectly maybe, then the error was thrown.
For test, run your app without the Parcel. Remove `import PlainDraggable ...` and add `<script>` element into your HTML code.
Or, use another bundler e.g. Webpack.

**Comment by romanzimnik** [comment]
Hi @anseki, thank you very much for your feedback. You have put me on the right track with the hint regarding webpack. I managed to solve my importing problem. Thanks for this!


For all who might also stumble upon a similar problem and come by here, I'll document my solution path once:

I removed parcel, installed plain-draggable via npm. I then use it in my custom js file to manipulate my SVGs. The import statement of plain-draggable is still present in my js-file and webpack does the bundling. The output is then simply integrated via `<script>` in the signle HTML-file I use for testing.
